templates/app-ts/tsconfig.json

```diff
@@ -1,22 +1,22 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "target": "ES2018",
+        "module": "CommonJS",
+        "moduleResolution": "node",
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
-        "target": "es6",
-        "moduleResolution": "node",
+        "strict": true,
         "sourceMap": true,
         "outDir": "dist",
         "baseUrl": ".",
         "paths": {
-            "*": [
-                "node_modules/*"
-            ]
+            "*": ["node_modules/*"]
         }
     },
     "include": [
+        "app.ts",
         "routes/**/*.ts",
         "plugins/**/*.ts",
-        "app.ts"
+        "test/**/*.ts"
```

Comment:
We don't need to include test files in the tsconfig.json that handles build for our app.

Reply:
Done